### DISCUSSION TITLE
Unexpected token '', ""... is not valid JSON

### DISCUSSION THREAD

**Discussion by IanGrainger** [johnlindquist/kit#1270] [Errors]
I got an error in a weird background window when I start my laptop today. Trying to run this install gives me the same error:

```
C:\Users\Me>npx @johnlindquist/install-kit@latest
Need to install the following packages:
  @johnlindquist/install-kit@1.2.12
Ok to proceed? (y) y
npm WARN deprecated har-validator@5.1.5: this library is no longer supported
npm WARN deprecated querystring@0.2.0: The querystring API is considered Legacy. new code should use the URLSearchParams API instead.
npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
undefined:1



SyntaxError: Unexpected token '', ""... is not valid JSON
    at JSON.parse (<anonymous>)
    at C.read (file:///C:/Users/Me/AppData/Local/npm-cache/_npx/3415c6bfffe36ec5/node_modules/@johnlindquist/kit-internal/dist/lowdb.js:1:4126)
    at async s.read (file:///C:/Users/Me/AppData/Local/npm-cache/_npx/3415c6bfffe36ec5/node_modules/@johnlindquist/kit-internal/dist/lowdb.js:1:590)
    at async db (file:///C:/Users/Me/AppData/Local/npm-cache/_npx/3415c6bfffe36ec5/node_modules/@johnlindquist/kit/core/db.js:43:5)
    at async file:///C:/Users/Me/AppData/Local/npm-cache/_npx/3415c6bfffe36ec5/node_modules/@johnlindquist/kit/share/auth-scriptkit.js:8:16

Node.js v20.1.0
npm notice
npm notice New patch version of npm available! 9.6.4 -> 9.6.7
npm notice Changelog: https://github.com/npm/cli/releases/tag/v9.6.7
npm notice Run npm install -g npm@9.6.7 to update!
npm notice

**Comment by johnlindquist** [comment]
@IanGrainger Check the json files in the ~/.kit/db folder. 

That "invalid json" error is almost always one of those files being invalid json. If you find which one, please let me know so I can know where to debug 🙏

You can quit the Kit app, remove all the files in the "db" folder, then the next time you open Kit it will regenerate the files and the error should go away.

**Reply by IanGrainger** [reply]
> @IanGrainger Check the json files in the ~/.kit/db folder.
> 
> That "invalid json" error is almost always one of those files being invalid json. If you find which one, please let me know so I can know where to debug 🙏
> 
> You can quit the Kit app, remove all the files in the "db" folder, then the next time you open Kit it will regenerate the files and the error should go away.

Thanks for the quick reply! I'd actually already uninstalled, removed all the files and reinstalled the last build. Which is now working. I should reinstall the latest I guess!

**Reply by guillermoarellano** [reply]
@johnlindquist This worked for me on Windows 11 Home edition. Thank you!
